Revised_read_jma

```diff
@@ -0,0 +1,26 @@
+            # Mark if this is a shortened line - does not contain 30 and 50 kts
+            # wind radius - set those to -1 to mark them as missing
+            if len(data) < 8:
+                self.max_wind_radius[i] = -1
+            else:
+                data[7] = int(data[7]) - int(int(data[7])/10000)*10000
```

Comment:
Try to adhere to the PEP8 code-styling guidelines.  Here putting spaces around operators is what is important but please take a look at the guidelines for other ideas.

Reply:
Ok, I will revise it.